### ISSUE TITLE
[java] AvoidCatchingThrowable can not detect the case: catch (java.lang.Throwable t)

### ISSUE THREAD

**Issue by wuchiuwong** [pmd/pmd#2439]
**Affects PMD Version:** 
6.22.0

**Rule:**
[AvoidCatchingThrowable](https://pmd.github.io/latest/pmd_rules_java_errorprone.html#avoidcatchingthrowable)

**Description:**
AvoidCatchingThrowable can not detect the case: <code>catch (java.lang.Throwable t)</code>.
The undetectable reason is that using <code>if (name.hasImageEqualTo("Throwable")) {</code> in pmd-java-6.22.0-sources.jar\net\sourceforge\pmd\lang\java\rule\errorprone\AvoidCatchingThrowableRule.java_24 to detect can't coverage this case.

**Code Sample demonstrating the issue:**

```java
catch (java.lang.Throwable t) {
    throw new javax.xml.rpc.ServiceException(t);
}

```

**Expected outcome:**
false-negative

**Running PMD through:**
CLI
